Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -21,7 +21,7 @@ First the Prometheus operator needs to be started in the cluster so it can watch
 A full explanation can be found in the [Prometheus operator repository on GitHub](https://github.com/prometheus-operator/prometheus-operator), but the quick instructions can be found here:
 
 ```console
-kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.40.0/bundle.yaml
+kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.71.1/bundle.yaml
```

Comment:
Could you confirm that you have tested this ceph monitoring guide with the new Prometheus version? It's possible we'll find other updates needed since it has been so long since it was updated.